src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -124,8 +125,8 @@ const ArticlePage = ({ pageData }) => {
     visuallyHiddenHeadline,
     headline: headings,
     subheadline: headings,
-    audio: articleMediaPlayer,
-    video: articleMediaPlayer,
+    audio: isAmp ? articleMediaPlayer : MediaLoader,
+    video: isAmp ? articleMediaPlayer : MediaLoader,
```

Comment:
This work is solely focused on canonical, so continuing to use the old media player for AMP for now.